AIRO/README.md

```diff
@@ -1,13 +1,21 @@
-# AIRO
-**AIRO: an ontology for AI risk management**
+# AIRO (AI Risk Ontology)
+
 AIRO represents AI risk according to the EU AI Act proposal and ISO 31000 standard series. 
-AIRO is available online at https://delaramglp.github.io/AIRODoc/
+
+This directory is moved to https://github.com/perma-id/w3id.org/tree/master/airo
+
 
 **Contact:**
+
 This ontology is created and maintained by:
+
 Delaram Golpayegani
+
 PhD Researcher
+
 ADAPT Centre, Trinity College Dublin, Dublin, Ireland
+
 delaram.golpayegani@adaptcentre.ie
+
 GitHub: delaramglp
```

Comment:
If you merge this suggestion as-is, all line breaks will be retained as desired.
```suggestion
## Contact

This ontology is created and maintained by:

Delaram Golpayegani  
PhD Researcher  
ADAPT Centre, Trinity College Dublin, Dublin, Ireland  
delaram.golpayegani@adaptcentre.ie  
GitHub: delaramglp
```